go-controller/pkg/ovn/egressfirewall.go

```diff
@@ -415,23 +395,30 @@ func (oc *Controller) createEgressFirewallRules(priority int, match, action, ext
 	if err != nil {
 		return fmt.Errorf("error executing find ACL command, stderr: %q, %+v", stderr, err)
 	}
+	sort.Strings(logicalSwitches)
 	for _, logicalSwitch := range logicalSwitches {
 		if uuids == "" {
-			_, stderr, err := util.RunOVNNbctl("--id=@acl", "create", "acl",
+			id := fmt.Sprintf("%s-%d", logicalSwitch, priority)
+			stdout, stderr, err := txn.AddOrCommit([]string{"--id=@" + id, "create", "acl",
 				fmt.Sprintf("priority=%d", priority),
-				fmt.Sprintf("direction=%s", types.DirectionFromLPort), match, "action="+action,
+				fmt.Sprintf("direction=%s", types.DirectionFromLPort), match, "action=" + action,
 				fmt.Sprintf("external-ids:egressFirewall=%s", externalID),
 				"--", "add", "logical_switch", logicalSwitch,
-				"acls", "@acl")
-			if err != nil {
-				return fmt.Errorf("error executing create ACL command, stderr: %q, %+v", stderr, err)
+				"acls", "@" + id})
+			if len(stdout) != 0 {
+				klog.V(5).Infof("Partially commiting commands from updateEgressFirewall")
```

Comment:
@JacobTanenbaum You can leave this out, because if you look at the AddOrCommit() code, you'll only ever get `stdout` when `err` is set. Plus the code already (more generically) prints a message when it does a partial commit.

Reply:
done